include/Arcade/Core/IDisplayModule.hpp

```diff
@@ -50,6 +51,16 @@ namespace Arcade {
                  *
                  */
                 virtual void changeGraphicLib() = 0;
```

Comment:
Add doc to say it is forbidden to call this in systems
and to use event instead

Reply:
done in #77